.rubocop.yml

```diff
@@ -57,6 +57,11 @@ Layout/SpaceAroundEqualsInParameterDefault:
 # are needed.
 Style/StringLiterals:
   EnforcedStyle: double_quotes
+  Exclude:
+    - "config/application.rb"
+    - "config/boot.rb"
+    - "config/environment.rb"
+    - "config/environments/*.rb"
```

Comment:
I assume you excluded those files to allow for single-quotes in the config files to keep the diff to the Rails-original sources minimal? If so, a comment stating as much probably would be a good idea, because I was about to write a comment asking why you flipped the quotes in those files. :)

Reply:
Yes, that's exactly the reason, because when updating rails it also generates new versions of these files, and I wanted to keep them as close to the original as possible to see what actually changed ... but yes, I'll add a comment here :+1: